src/server/Dockerfile

```
@@ -0,0 +1,20 @@
+FROM continuumio/miniconda3
+
+COPY . .
```

separate copying the requirements.txt and other files

```suggestion
COPY ./requirements.txt .
```